scss/components/_callout.scss

```diff
@@ -26,6 +26,14 @@ $callout-margin: 0 0 1rem 0 !default;
 /// @type Number
 $callout-padding: 1rem !default;
 
+/// Default inner padding for small callouts.
+/// @type Number
+$callout-small-padding: 0.5rem !default;
+
+/// Default inner padding for large callouts.
+/// @type Number
+$callout-large-padding: 3rem !default;
+
```

Comment:
What about a map of sized instead, like for buttons:

```scss
/// Sizes for Callout paddings.
/// @type Map
$callout-sizes: (
  small: 0.5rem,
  default: 1rem,
  large: 3rem,
) !default;
```

This is more customizable, and more or less the way all components manage their modifiers (sizes, colors).

Reply:
Sounds good to me.  I took a stab at it.